utility/src/main/java/org/oppia/util/firebase/CrashlyticsWrapper.kt

```diff
@@ -0,0 +1,27 @@
+package org.oppia.util.firebase
+
+import com.google.firebase.crashlytics.FirebaseCrashlytics
+import javax.inject.Inject
+import javax.inject.Singleton
+
+/** Controller for providing custom crash reporting to Firebase Crashlytics */
+@Singleton
+open class CrashlyticsWrapper @Inject constructor(
+  private var firebaseCrashlytics: FirebaseCrashlytics
+) {
+
+  /** Logs a custom non-fatal exception to Firebase Crashlytics */
+  open fun logException(exception: Exception) {
+      firebaseCrashlytics.recordException(exception)
+  }
+
+  /** Logs a custom log message which can be put alongside a crash report to Firebase Crashlytics */
+  open fun logMessage(message: String) {
+    firebaseCrashlytics.log(message)
+  }
+
+  /** Sets up a user identifier which is attached in every crash report to Firebase Crashlytics */
+  open fun setUserIdentifier(identifier: String) {
```

Comment:
One of the intents of this project was to not associate logs with an individual user--how is this meant to be used?

Reply:
Okay, so both the logMessage and setUserIdentifier are kept for future usages. 
One use case of setUserIdentifier can be providing the countries of our users. We can then possibly see if our app is crashing in a particular region.

Reply:
In general, we should only implement what we specifically need for a particular piece of the project or in anticipation of work soon-to-be done. We can always add new functionality later once it's determined to be needed.